src/db/migrations/20230703010356_add-profile-table.js

```diff
@@ -0,0 +1,27 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+export function up(knex) {
+  return knex.schema
+    .createTable('profiles', table => {
+      table.increments('id').primary()
+      table.integer('onerep_profile_id').references('subscribers.onerep_profile_id').nullable()
```

Comment:
Hmm... Do we want to store OneRep profiles if we don't know which user they belong to? (Referring to the `.nullable()` here.)